krm-functions/lib/interface/v1alpha1/interface.go

```diff
@@ -216,6 +234,7 @@ func (r *itfce) getStringValue(fields ...string) string {
 
 // setNestedField is a generic utility function that sets a string on
 // a string slice representing the path in the yaml doc
+/*
```

Comment:
if we don't need these functions anymore, we should just remove them.

Reply:
My suggestion is to use them as this is common boiler plate code.

Reply:
fixed